src/linear_algebra/determinant.lean

```diff
@@ -400,3 +400,12 @@ by rw [basis.det_reindex, function.comp.assoc, e.self_comp_symm, function.comp.r
 lemma basis.det_map (b : basis ι R M) (f : M ≃ₗ[R] M') (v : ι → M') :
   (b.map f).det v = b.det (f.symm ∘ v) :=
 by { rw [basis.det_apply, basis.to_matrix_map, basis.det_apply] }
+
+/-- This lemma is useful because it enables us to see `matrix.det` as (the coercion of) an
+`alternating_map` on the module `ι → R`. -/
```

Comment:
Is that different to what `matrix.det_row_multilinear` is?

Reply:
Maybe simply replacing
```lean
abbreviation det (M : matrix n n R) : R :=
det_row_multilinear M
```
with
```lean
abbreviation det : matrix n n R → R :=
(det_row_multilinear : alternating_map R _ _ n)
```
achieves what you need here, then `dunfold matrix.det` introduces such an alternating map without an annoying lambda.

Reply:
Oh, thanks. How did I miss `matrix.det_row_multilinear`; it should indeed be sufficient for my purposes.

I'll come back to this in a few days once I've finished the other parts of what I need.

Reply:
Perhaps it should be renamed `det_row_alternating`; I didn't change the name when I upgraded it to an alternating map.

I think the lemmas in this PR are still worth having, maybe just this one in the other direction and without the misleading docstring motivation

Reply:
Agreed on all points. I have attempted to implement these suggestions.